godtools/App/Share/ColorPalette.swift

```diff
@@ -61,6 +67,14 @@ enum ColorPalette {
         return UIColor(red: 128 / 255, green: 130 / 255, blue: 132 / 255, alpha: 1.0)
     }
     
+    private func getMediaColor() -> UIColor {
+        return UIColor(red: 230 / 255, green: 230 / 255, blue: 230 / 255, alpha: 1.0)
+    }
+    
+    private func getPrimaryTextColor() -> UIColor {
+        return UIColor(red: 90 / 255, green: 90 / 255, blue: 90 / 255, alpha: 1.0)
```

Comment:
`.primaryTextColor` is the same as `.gtGrey`.  We could just return gtGrey in the primaryTextColor definition, instead of defining the same color twice.

Reply:
Hey @rachaelblue the color naming gets a little tricky to me. Because maybe some UI elements share a color, however, we might want to create a specific color tied to different types of UI elements.  That way changing a color only effects those specific UI elements, not all elements that share that color.  Does that make sense?  Does ```gtGrey``` get applied to anything other than text and anything that might not be primary text?  I'm not sure if ```primaryTextColor``` is even a good name because what elements would be considered as primary text?  Maybe this is where it's best to have a designer standardize the app color theme that we just mimic?  What are your thoughts here and in other apps you've worked on how have you solved those questions?

Reply:
Other than those questions, I think returning ```gtGrey``` from ```primaryTextColor``` is a good idea.

Reply:
Totally get ya, this is always a conundrum.  I tend to steer away from naming colors for places they're used, like `primaryTextColor`, because so often the color ends up changing in some areas but not others, and then the name is no longer accurate, but it rarely gets corrected because the color itself is still in use.  Also, it's harder to figure out what the color is _supposed_ to be when it's named after the area where it's used.  And then, more often than not, there's just a graveyard of colors that may or may not be used, but no one touches them because no one knows which we still need.

I think, ideally, a designer would standardize the app colors, and we'd name them things like gtGrey, gtBlue, etc. There shouldn't be _that_ many colors (and there really aren't now, either), and they shouldn't really change unless the entire app is being rebranded.  And if there's only a few core colors to choose from, I think the `primaryTextColor` naming scheme becomes unnecessary.  If you need a little more variation, names like `gtPrimaryGrey` and `gtSecondaryGrey` help distinguish without getting too specific.

Reply:
Hey @rachaelblue this is good input because I would have created a graveyard of colors.  Makes more sense to keep naming less specific.